[DOCS] Consolidates KSPM FAQs into one page (#2625)
elastic/security-docs: docs/cloud-native-security/kspm-faq.asciidoc

[[kspm-faq]]
== KSPM FAQ

> Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.

Reviewer: ```suggestion
Frequently asked questions about the Kubernetes Security Posture Management (KSPM) integration and features.
```

[discrete]
== General KSPM FAQ

*Can I evaluate the security posture of my Amazon EKS clusters?*
Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.

*How often is my cluster’s security posture evaluated?*

> Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.

Reviewer: What do you think about combining the "General KSPM FAQ" section with the Cluster enrollment FAQ" section? Both sections seem to address common questions about clusters, so maybe you could group them together under a common section titled "Cluster enrollment and security", "Cluster monitoring", or just "Clusters".

> Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.

Reviewer 2: +1 on combining these sections per @nastasha-solomon. For a relatively short page, it probably doesn't need to be so heavily segmented and granular.

Also a possible edit for tone:
```suggestion
Clusters are evaluated when you deploy a KSPM integration, and every four hours after that.
```

[discrete]

> === Findings page FAQ

Reviewer 2: +++

> *Are all the findings on the Findings page current?*

Reviewer: ```suggestion
*Are all the findings current?*
```

Yes. Only the most recent findings appear on the Findings page.

*Can I build custom visualizations and dashboards that incorporate findings data?*

> Yes. You can use {kib}'s custom visualization capabilities with findings data. Current findings data can be accessed using the `logs-cloud_security_posture.findings_latest-*` index pattern, while historical findings data can be accessed using the `logs-cloud_security_posture.findings-*` index pattern. To learn more, refer to {kibana-ref}/dashboard.html[Dashboards and visualizations].

Reviewer: Do you need the second half about where to find current and historical data? Or is that information more relevant to the next question? 

```suggestion
Yes. You can use findings data to build {kibana-ref}/dashboard.html[custom visualizations] in {kib}. 
```

*Can I view past findings?*

> Yes. Historical findings data is accessible using the `logs-cloud_security_posture.findings-*` index pattern.

Reviewer 2: ❤️❤️❤️ the idea of a list for this kind of info. Easily scannable, neat and tidy.

Also, it seems like this FAQ and the previous one could maybe be combined, or each one revised to be more focused -- that looks like what @nastasha-solomon is getting at in the previous comment. They're both answering the same question, how do I find and visualize current and past findings?

[discrete]
=== Benchmark rules FAQ

*Do benchmark rules support multiple Kubernetes deployment types?*
Yes. There are different sets of benchmark rules for self-managed and EKS-managed deployments. Refer to <<get-started-with-kspm,Get started with KSPM>> for more information about setting up each deployment type.

*How often are my resources evaluated against benchmark rules?*
Resources are fetched and evaluated against benchmark rules when a KSPM integration is deployed, and every four hours after that.

*Can I configure the KSPM integration's fetch cycle?*
No, the four-hour fetch cycle is not configurable.

*Can I contribute to the CSP ruleset?*
You can't directly edit benchmark rules. The rules are defined https://github.com/elastic/csp-security-policies[in this repository], where you can raise issues with certain rules. They are written in https://www.openpolicyagent.org/docs/latest/policy-language/[Rego].

*How can I tell which specific version of the CIS benchmarks is in use?*

> Refer to the `rule.benchmark.name` and `rule.benchmark.version` fields for documents in these datastreams: `logs-cloud_security_posture.findings-default` and `logs-cloud_security_posture.findings_latest-default`

Reviewer: Made this suggestion to visually break up the last part of the answer which looks very stylistically similar.

> Refer to the `rule.benchmark.name` and `rule.benchmark.version` fields for documents in these datastreams: `logs-cloud_security_posture.findings-default` and `logs-cloud_security_posture.findings_latest-default`

Reviewer 2: ++
